slackEvents.php

```diff
@@ -493,7 +493,11 @@ public function event_selection($channel_id, $trigger_id) {
         $app_id = $app_infos->user_id;
         $members = $this->api->conversations_members($channel_id);
 
-        if (!in_array($app_id, $members)) {
+        if (is_null($members) || !in_array($app_id, $members)) {
+            $text = is_null($members) ?
+                    "Vous avez probablement essayé d'excuter cette commande depuis une conversation privée, ce qui n'est pas supporté. Si vous pensez qu'il s'agit d'une erreur, vous pouvez contacter #c-outils-numeriques" :
```

Comment:
Maybe we can change the message to something less related to our NGO ?

> [...] vous pouvez contacter votre administrateur.

Or something like that?